proteus/femIntegrals.h

```diff
@@ -738,9 +738,9 @@ extern void updateGlobalJacobianFromInteriorElementBoundaryFluxJacobian_dense(
   int *elementBoundaryElements,
   int *elementBoundaryLocalElementBoundaries,
   int *nFreeDOF_element_r,
-  int *nFreeDOF_element_u,
   int *freeLocal_r,
   int *freeGlobal_r,
+  int *nFreeDOF_element_u,
```

Comment:
Note @manuel-quezada I also changed the order of these arguments to be consistent across python and C. In Python we are always using (all r stuff),(all u stuff) (e.g. nFreeDOF_element_r, freeLocal_r, freeGlobal_r, nFreeDOF_element_u,...), but the order was different in C, so the wrappers would have to catch that re-rordering. With this change there is not re-ordering going on.